intake_stac/tests/test_catalog.py

```diff
@@ -35,7 +39,12 @@ def pystac_cat():
 
 @pytest.fixture(scope='module')
 def pystac_col():
-    return pystac.Collection.from_file(col_url)
+    col = pystac.Collection.from_file(col_url)
+    # fix some linkrot
```

Comment:
One of the links in this file is unfortunately valid, so we hit an error when PyStac tried to load all the children. We're pointing to the tag on GitHub, so we can't change the source. Probably OK to just fix it here.

Reply:
Yeah, I've noticed a lot of existing test catalogs don't worry so much about the validity of asset links, whereas with intake-stac we do want to test against real assets. maybe we embed a couple test catalogs in this repository or update and use https://github.com/stac-utils/stac-examples/issues/1 ?

Reply:
Note that `https://landsat-stac.s3.amazonaws.com/landsat-8-l1/catalog.json` will be deprecated on July 1 (https://registry.opendata.aws/landsat-8/)